res/sections/16-Les21032017.tex

```diff
@@ -220,13 +221,27 @@ \subsection{Biometria}
 Chi sei e cosa fai, è suscettibile ad errori.
 
 \begin{itemize}
-\item \textbf{False Rejection Rate:} anche detto \textbf{falso negativo}, un 
-reject che non dovrebbe esserci stato;
-\item \textbf{False Acceptance Rate:} anche detto \textbf{falso positivo}, un 
-accept quando non avrebbe dovuto esserci;
-\item \textbf{Failure to Enroll Rate:} numero di utenti che non sono riusciti a
-registrarsi correttamente.
+\item \textbf{False Rejection Rate} (FRR): anche detto 
```

Comment:
esatto! non so se si dovrebbero cambiare allora le definizioni in qualcosa del tipo 
> \textbf{False Rejection Rate} (FRR): è la percentuale di \textbf{falsi negativi}, oovero quando si è verificato un reject che non dovrebbe esserci stato;